src/dragDrop.js

```diff
@@ -0,0 +1,114 @@
+// Initials tasks
+/* eslint-disable import/no-mutable-exports,  */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-loop-func */
+
+import { clearAll } from './index.js';
+import addTask from './addTask.js';
+
+export let tasks = [];
+
+if (localStorage.getItem('tasks')) {
+  const getList = JSON.parse(localStorage.getItem('tasks'));
+  tasks = getList;
+}
+
+// Add Drag-start and drag-End listeners --- Add Css Class
+const addListeners = (elements) => {
+  elements.forEach((item) => {
+    item.addEventListener('dragstart', () => {
+      item.classList.add('dragging');
+
+      item.addEventListener('dragend', () => {
+        item.classList.remove('dragging');
+      });
+    });
+  });
+};
+
+/// Code for local Storage Save.
+
+export const saveLocalstorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Help save any changes in real time (Save position after DragDrop, etc...)
+//----------------------
```

Comment:
I suggest you make a function to evaluate if it's existing local storage on the browser before trying to save data in order to prevent script crash.